docs/oauth2-oidc/client-credentials.mdx

```diff
@@ -36,7 +36,7 @@ as you would not write your own SHA512 library.
 
 The first and most common mechanism for authenticating a client in the Client Credentials flow is Basic Authentication. In this
 mechanism, the client sends its client ID and client secret as part of the Authorization header in an HTTP request. The
-Authorization header is formed by concatenating the urlencoded client ID and client secret, separated by a colon, and base64
+Authorization header is formed by concatenating the url_encoded client ID and client secret, separated by a colon, and base64
 encoding the result.
```

Comment:
I'm not sure that this change makes a significant improvement. The suggestion also seems to have a syntax error (";" instead of ":").  Can we leave it as it is? Or maybe I can just change url_encoded to URL-encoded (as that is a bit ugly)?

Reply:
Ah, didn't realise you could edit the suggestion. I noticed there was also a missing backtick character, so I just added that as well.